packages/config/config/devices/0x0154/flow_stop_2.json

```diff
@@ -0,0 +1,26 @@
+// Popp & Co Flow Stop
+// Version 2
+{
+	"manufacturer": "Popp & Co",
+	"manufacturerId": "0x0154",
+	"label": "Flow Stop 2",
+	"description": "",
```

Comment:
```suggestion
```

Reply:
I'm guessing you can ignore lint failures? ,as it failed when I removed description.

Reply:
The model # is POPE009501. That should be the label. The description is Flow Stop.

Reply:
That's the other one ;)

Reply:
It looks like that's version 1? Version 2 is POPP-701479-862